lib/model/model.go

```diff
@@ -1467,18 +1467,32 @@ func sendIndexTo(minSequence int64, conn protocol.Connection, folder string, fs
 }
 
 func (m *Model) updateLocalsFromScanning(folder string, fs []protocol.FileInfo) {
+	// If we're running this code it's because our node last add/moved/changed
+	// this file.  So we set LastModBy to our devices friendly name
+	for i := range fs {
+		fs[i].LastModBy = m.deviceName
+	}
 	m.updateLocals(folder, fs)
 
 	m.fmut.RLock()
 	folderCfg := m.folderCfgs[folder]
 	m.fmut.RUnlock()
 
 	// Fire the LocalChangeDetected event to notify listeners about local updates.
-	m.localChangeDetected(folderCfg, fs)
+	localChange := true
+	m.diskChangeDetected(folderCfg, fs, localChange)
```

Comment:
I actually only put it cause I thought you guys would want it for clarity.  I prefer brevity myself as long as it's documented somewhere, and I assume most programmers use an IDE to read code so they can just jump to the function declaration from that line to see what the variable is used for.